updated whitepaper link to resolve https://github.com/keylime/keylimegithub.io/issues/14

atothRedHat opened the pull request:
updated the white paper link to point to  https://github.com/keylime/keylime/blob/master/doc/tci-acm.pdf instead of outdated LL hosted link

Commit:
updated whitepaper link to resolve https://github.com/keylime/keylime.github.io/issues/14

lukehinds reviewed (changes requested):
Could you remove the google viewer URL part and just link to https://github.com/keylime/keylime/raw/master/doc/tci-acm.pdf

atothRedHat commented:
without the google viewer it just dumps you into the github location, I didn't think that is haow we wanted it displayed when the link is selected. If you what it to show in the github page I'll change it. (follow these links (raw then blob) to see how it is displayed without the google viewer: https://github.com/keylime/keylime/raw/master/doc/tci-acm.pdf    and    https://github.com/keylime/keylime/blob/master/doc/tci-acm.pdf)

atothRedHat commented:
one other option is to move the pdf into this repo but not sure we want things strewn about through multiple repos.

lukehinds commented:
Got you now, I think raw is best: https://github.com/keylime/keylime/raw/master/doc/tci-acm.pdf

That way if someone has a default PDF reader set in their browser, that's what will get used.

atothRedHat commented:
ack, will update and repost

Commit:
updated whitepaper link to resolve https://github.com/keylime/keylime.github.io/issues/14
Using raw location which will download the pdf to the requestors machine when the link is selected.

atothRedHat commented:
updated to raw link

lukehinds reviewed (approved):
Thanks!
